src/main/kotlin/no/nav/familie/tilbake/api/dto/FagsakDto.kt

```diff
@@ -1,17 +1,28 @@
 package no.nav.familie.tilbake.api.dto
 
+import no.nav.familie.tilbake.behandling.domain.Behandlingsstatus
+import no.nav.familie.tilbake.behandling.domain.Behandlingstype
 import no.nav.familie.tilbake.behandling.domain.Fagsaksstatus
 import no.nav.familie.tilbake.behandling.domain.Ytelsestype
 import no.nav.familie.tilbake.integration.pdl.internal.Kjønn
 import java.time.LocalDate
+import java.util.UUID
 
 data class FagsakDto(val eksternFagsakId: String,
                      val status: Fagsaksstatus,
```

Comment:
Eg ser at eg har kopiert denne frå ba-sak, men har me i familie-tilbake ein fagsakstatus? Trenger me det?

Reply:
Jeg tror vi trenger det

Reply:
Til kva? Det er jo ikkje tilbakekreving som styrer status på fagsaken? Holder det ikkje berre for fagssystemet å vite om det finnes ein åpen behandling?

Reply:
Jeg tror det kan fjernes senere hvis vi ikke bruker det